tests/analysis/adaptation/test_adaptation.py

```diff
@@ -66,7 +30,26 @@ def test_run_cost(
         assert isinstance(_cost_gdf, GeoDataFrame)
         assert all(
             [
-                f"costs_{_option.id}" in _cost_gdf.columns
+                f"{_option.id}_cost" in _cost_gdf.columns
                 for _option in _adaptation.adaptation_collection.adaptation_options
             ]
         )
+
+    def test_run_benefit(
```

Comment:
what are you testing? define in the method. 
Why is the test not checking the values of the columns?

Reply:
Adjusted